lib/layout.js

```diff
@@ -229,7 +229,9 @@ const PodiumLayout = class PodiumLayout {
                 // if this is a proxy request then no further work should be done.
                 if (incoming.proxy) return;
 
-                setAtLocalsPodium(res, 'context', incoming.context);
+                // set "incoming" on res.locals.podium
+                objobj.set('locals.podium', incoming, res);
```

Comment:
are we moving away from using `setAtLocalsPodium`?

Reply:
Yes. `res.locals` is a very Express thingy and since we are making Podium http framework agnostic we should also try to not have such methods which is Express related. We already do this in `@podium/podlet`.